.github/workflows/ci-backend.yml

```diff
@@ -0,0 +1,76 @@
+name: Backend Test & Lint
+
+on:
+  push:
+    paths:
+      - 'backend/**'
```

Comment:
See comment to frontend: docker-compose file should be included here, I believe.